search-enterprise/latest/config/transactional.md

```diff
@@ -0,0 +1,382 @@
+---
+title: Transactional metadata query
+---
+
+Alfresco Content Services supports the execution of a subset of the CMIS Query Language (CMIS QL) and Alfresco Full Text Search (AFTS) queries directly against the database. Also, the noindex subsystem supports queries only against the database. This collection of features is called transactional metadata query (TMDQ).
+
+TMDQ supports use cases where eventual consistency is not the preferred option.
+
+The Elasticsearch subsystem is eventually consistent. A change can take any length of time to be reflected in the index, ranging from a few seconds to several minutes. Elasticsearch indexes the metadata and the content of each updated node, in the order in which the nodes were last changed. The indexing components will try to index information about nodes as fast as possible, but content indexing is likely to be limited by the time needed to extract text from the files and all indexing will be affected by the rate at which the nodes are being changed.
+
+Some queries can be executed both transactionally against the database or with eventual consistency against the Solr index. Only queries using the AFTS or CMIS query languages can be executed against the database. The Lucene query language cannot be used against the database whereas, `selectNodes` (XPATH) on the Java API always goes against the database, walking and fetching nodes as required.
+
+Improvements to tracking results in less lag to metadata indexing. Metadata updates are impacted less by content indexing or the bulk updates to PATH for `move`, `rename`, `link` and, `unlink` operations.
```

Comment:
I don't understand what this paragraph is getting at - perhaps it's in reference to a particular release? I'd suggest removing it completely.

Reply:
Agree. Elasticsearch docs should not have any references to tracking.

Reply:
Removed.

Reply:
Tracking is not in the docs rechecked.

Reply:
It is still not there.